### PR TITLE
Added deck and logic for displaying the deck

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Added the `post_deck` ACF field and added logic for displaying it on single posts. Also added a hook for replacing the excerpt with the deck when it's available.

**Motivation and Context**
We want to be able to display a deck at the top of post updates.

**How Has This Been Tested?**
Changes are available for review in DEV.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: dev/acf-export.json [hunk 1]
```diff
@@ -118,25 +118,7 @@
...
-                    "width": "",
-                    "class": "",
-                    "id": ""
-                },
-                "message": "<div class=\"notice notice-info inline\"><p><strong>Please note:<\/strong> The Home Icon Menu location does not support nested menu items.  Any nested menu items added to this menu will not be displayed on the homepage.<\/p><\/div>",
-                "new_lines": "wpautop",
-                "esc_html": 0
-            }
-        ],
+        "fields": [],
```
Comment (cjg89) [inline dev/acf-export.json:121]: Did the existing message field in this group get wiped out somehow?

File: dev/acf-export.json [hunk 2]
```diff
@@ -722,6 +704,13 @@
                     "value": "page"
                 }
             ],
+            [
+                {
+                    "param": "post_type",
```
Comment (cjg89) [inline dev/acf-export.json:709]: The page header field group shouldn't be present for single posts (this is a custom change for this theme)

File: dev/acf-export.json [hunk 3]
```diff
@@ -738,5 +727,47 @@
...
+    },
+    {
+        "key": "group_5e8777494f9e9",
+        "title": "Post Fields",
+        "fields": [
+            {
+                "key": "field_5e87774dce0c5",
+                "label": "Deck",
+                "name": "post_deck",
+                "type": "wysiwyg",
```
Comment (cjg89) [inline dev/acf-export.json:739]: We may want to consider simplifying the WYSIWYG editing tools for this field--we probably don't want images and such to be added here.  I believe Today is using a custom WYSIWYG toolbar set for their "deck" field: https://github.com/UCF/Today-Child-Theme/blob/master/includes/config.php#L233-L249

File: single.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+$deck = get_field( 'post_deck' );
+?>
+
+<?php get_header(); the_post(); ?>
+
+<article class="<?php echo $post->post_status; ?> post-list-item">
+	<div class="container mt-4 mt-sm-5 mb-5 pb-sm-4">
+		<?php if ( ! empty( $deck ) ) : ?>
+		<div class="lead mb-3">
```
Comment (cjg89) [inline single.php:14]: Can we change the `mb-3` here to `mb-4`?

File: dev/acf-export.json (hunk 3 above)
Comment (jmbarne3) [reply dev/acf-export.json:750]: Screen Shot 2020-04-06 at 10 58 20 AM

These are the controls currently available. Do we need to reduce it down further? Images currently can't be uploaded.

**Comment by jmbarne3** [comment]
Resolves #21 and #27

**Pull request merged**